cfgov/unprocessed/js/modules/util/data-hook.js

```diff
@@ -60,8 +60,22 @@ function contains( element, value ) {
   return values.indexOf( value ) > -1 ? true : false;
 }
 
+/**
+ * @param {string} behavior
+ *  Behavior type used to find the element within the dom.
+ * @returns {HTMLNode} if it exists in the dom, null otherwise.
+ */
+function get( behavior ) {
```

Comment:
I'd still mention the element... buuuut I'm going on vacation and won't be checking GH for about 9 days. In any case, I look at it like this --> `add`, `remove`, and `contains` are all talking about values inside the data-hook and follow the classList API. By that logic, dataHook.get/dataHook.find would be looking for the value inside the data-hook. However, by retrieving the parent element this method is going the other direction and is talking about outside of the data-hook, so that difference is worth noting to prevent confusion that get/find would be returning the data-hook value.